lib/config.js

```diff
@@ -3,27 +3,36 @@
  *
  * Copyright (c) 2012-2015 Digital Bazaar, Inc. All rights reserved.
  */
-var config = require('bedrock').config;
+var bedrock = require('bedrock');
+var c = bedrock.util.config.main;
+var cc = c.computer();
+var config = bedrock.config;
 var path = require('path');
 
-config.server = {};
+c.setDefault('server', {});
```

Comment:
It could be reverted I think.  It was an attempt to consider how to handle some use cases where you would want to setup some extra server config keys before loading this module.  That may just be nonsense though.  Probably better to just have a policy you will reset your modules namespace.  If special use cases are needed then use some other mechanism.